### PR TITLE
Update install scripts

### PR DESCRIPTION
**Pull request by stdevMac**
## Changes:
- Updated installation scripts for docker and docker-compose.

## Types of changes

- [x] Refactoring (no functional changes, no api changes)

## Testing
Tested manually at:
- Ubuntu: 20.04 and 22.04
- Debian: 10 and 11
- Arch
- Amazon Linux 2

### CODE REVIEW

File: templates/setup/linux/docker/debian_10.sh [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
-sudo apt-get install -y docker-ce docker-ce-cli containerd.io
-
-# Install docker-compose
-
-DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
-mkdir -p $DOCKER_CONFIG/cli-plugins
-curl -SL "https://github.com/docker/compose/releases/download/v2.6.1/docker-compose-$(uname -s)-$(uname -m)" -o $DOCKER_CONFIG/cli-plugins/docker-compose
-chmod +x $DOCKER_CONFIG/cli-plugins/docker-compose
\ No newline at end of file
+sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
```
Comment (AntiD2ta) [inline templates/setup/linux/docker/debian_10.sh:23]: Is this `docker-compose` or `docker compose`? I'm not sure if the apt package is `docker compose`. We want `docker compose` as is now the correct and latest plugin. That's why we were using the above instructions to install it

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Are you installing `docker-compose` or `docker compose`? We want `docker compose`. Apart from that everything looks good. I will request changes for you to reply this and fix the Changelog

File: templates/setup/linux/docker/debian_10.sh (hunk 1 above)
Comment (stdevMac) [reply templates/setup/linux/docker/debian_10.sh:23]: [Here](https://docs.docker.com/engine/install/debian/#install-using-the-repository), at the docker documentation, you will see that the way to install and get the plugin `docker compose` is installing `docker-compose-plugin`.

**Review by AntiD2ta — APPROVED** [review]
Approved!

**Pull request merged**